enos/enos-scenario-replication.hcl

```diff
@@ -677,9 +677,9 @@ scenario "replication" {
     }
   }
 
-  step "write_test_data_on_primary" {
-    description = global.description.verify_write_test_data
-    module      = module.vault_verify_write_data
+  step "verify_secrets_engines_on_primary" {
```

Comment:
These changes will need to be applied to `dr_/pr_` replication scenarios, now that those are merged in